label_studio/organizations/middleware.py

```diff
@@ -16,78 +14,12 @@ def __init__(self, get_response):
 
     def __call__(self, request):
         org = Organization.objects.first()
+        user = request.user
+        if user and user.is_authenticated and user.active_organization is None:
+            user.active_organization = org
+            user.save(update_fields=['active_organization'])
         if org is not None:
```

Comment:
@triklozoid why didn't you remove this?

Reply:
I want to do it in next release, when all users will get their active_organizations. 

In out case we can remove it now I think, but for some more complicated things with more than one backend running simultaneously it can be good thing to not break something during deploy.